api/src/iot/tests/test_import_utils.py

```diff
@@ -324,19 +324,20 @@ def test_import_sensor_other_type(self, sensor_data):
         import_utils.import_sensor(sensor_data, owner)
         assert self.actual == [dict(self.expected, type="Midi-chlorian teller")]
 
-    def test_import_sensor_region(self, sensor_data):
+    def test_import_sensor_regions(self, sensor_data):
         # check that we can import as "mobile" sensor
         owner = import_utils.import_person(sensor_data.owner)
-        sensor_data.location = import_utils.Region("Centrum")
+        sensor_data.location = import_utils.Regions("Centrum,Oost")
```

Comment:
Wat als een gebruiker een komma gebruikt maar juist niet zou willen splitsen? Bijvoorbeeld "Indische Buurt, Oost"

Reply:
Er wordt inderdaad niet ge-escaped. Maar er is wel de mogelijkheid om een validatie te bouwen in IPROX zodat men geen ; mag gebruiken (mobiele sensoren kunnen niet met de bulk verwerking gemeld worden)